bin/vmpooler

```diff
@@ -9,6 +9,7 @@ redis_port = config[:redis]['port']
 redis_password = config[:redis]['password']
 redis_connection_pool_size = config[:redis]['connection_pool_size']
 redis_connection_pool_timeout = config[:redis]['connection_pool_timeout']
+redis_reconnect_attempts = config[:redis]['reconnect_attempts'] || 10
```

Comment:
This should ideally be in vmpooler.rb so we can evaluate the environment for parameters via env, and let the default be supplied there. We can then reference the value in config and it will be our default, the env supplied value, or the user supplied value in the config file depending on what is provided, and we can rely on the evaluation that happens when generating config.
Similar to something like this https://github.com/puppetlabs/vmpooler/blob/a35d66c606c7fc9f55dce71447f1465fec5830b6/lib/vmpooler.rb#L68